packages/base/any/kernels/3.16-lts/patches/driver-igb-netberg-aurora.patch

```diff
@@ -0,0 +1,599 @@
+diff -Nu a/drivers/net/ethernet/intel/igb/bcm_phy.c b/drivers/net/ethernet/intel/igb/bcm_phy.c
```

Comment:
These header definitions are already available in include/linux/brcmphy.h. 
54616 support is already available in e1000_82575 and e1000_hw. 

What is needed with this new file?